### DISCUSSION TITLE
Can FEATURE_FLAGS be enabled at will?

### DISCUSSION THREAD

**Discussion by srinisubramanian** [apache/superset#19479] [Q&A / Help] [answered]
Assume a running instance of Superset 1.4.1.  Is it ok to enable FEATURE_FLAGS that are now required in a running system?  After enabling do we need to run any commands or just restart?

**Comment by villebro** [comment, marked as answer]
Some feature flags may require running `superset init` after being enabled, but generally you can just enable feature flags at will. Keep in mind, however, that stability of features that are disabled by default is not guaranteed. If you have questions regarding a specific feature flag I'm happy to give you my best estimate regarding its stability.

**Reply by srinisubramanian** [reply]
For now I have enabled  DASHBOARD_RBAC, VERSIONED_EXPORT and ENABLE_TEMPLATE_PROCESSING.

I plan on only using those that are marked as "Stable" here https://github.com/apache/superset/blob/master/RESOURCES/FEATURE_FLAGS.md
